src/utils/utils.ts

```diff
@@ -620,3 +620,15 @@ export function stringClamp(value: string, def: number, min: number, max: number
     const nanToNull = (nr: number): null | number => (isNaN(nr) ? null : nr)
     return clamp(nanToNull(parseInt(value)) ?? def, min, max)
 }
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export function stringify(value: any): string {
+    switch (typeof value) {
+        case 'string':
+            return value
+        case 'undefined':
```

Comment:
I think `JSON.stringify(undefined)` returns "undefined" already

Reply:
It in fact returns `undefined` instead of `"undefined"`, which caught me off guard for a bit 😅

Reply:
ha, damn